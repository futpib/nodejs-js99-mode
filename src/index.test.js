
import test from 'ava';

import {parse} from '../';

test(t => {
	t.truthy(parse);
});
