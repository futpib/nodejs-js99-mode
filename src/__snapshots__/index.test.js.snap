// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[anonymous] 1`] = `
Object {
  "__clone": [Function],
  "comments": Array [],
  "end": 17,
  "loc": SourceLocation {
    "end": Position {
      "column": 17,
      "line": 1,
    },
    "start": Position {
      "column": 0,
      "line": 1,
    },
  },
  "program": Node {
    "body": Array [
      Node {
        "declaration": Node {
          "end": 17,
          "extra": Object {
            "raw": "42",
            "rawValue": 42,
          },
          "loc": SourceLocation {
            "end": Position {
              "column": 17,
              "line": 1,
            },
            "start": Position {
              "column": 15,
              "line": 1,
            },
          },
          "start": 15,
          "type": "NumericLiteral",
          "value": 42,
        },
        "end": 17,
        "loc": SourceLocation {
          "end": Position {
            "column": 17,
            "line": 1,
          },
          "start": Position {
            "column": 0,
            "line": 1,
          },
        },
        "start": 0,
        "type": "ExportDefaultDeclaration",
      },
    ],
    "directives": Array [],
    "end": 17,
    "loc": SourceLocation {
      "end": Position {
        "column": 17,
        "line": 1,
      },
      "start": Position {
        "column": 0,
        "line": 1,
      },
    },
    "sourceType": "module",
    "start": 0,
    "type": "Program",
  },
  "start": 0,
  "type": "File",
}
`;
